webapp/src/dogma/features/auth/authSlice.ts

```diff
@@ -17,14 +17,13 @@
 import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
 import axios from 'axios';
 import qs from 'qs';
-import { history } from 'dogma/store';
 import { UserDto } from 'dogma/features/auth/UserDto';
-import { getSessionId, goToLoginPage, removeSessionId } from 'dogma/features/auth/Authorized';
 import { HttpStatusCode } from 'dogma/features/api/HttpStatusCode';
+import { setSessionId, removeSessionId, getSessionId } from 'dogma/features/auth/util';
 
 const getUser = createAsyncThunk('/auth/user', async () => {
   // TODO(ikhoon): Just use fetch API?
-  const response = await axios.get('/api/v0/users/me');
+  const response = await axios.get(`${process.env.NEXT_PUBLIC_HOST}/api/v0/users/me`);
```

Comment:
How about settings the base URI to the global configurations?
https://axios-http.com/docs/config_defaults